visualizer/actionButton.js

```diff
@@ -12,6 +12,10 @@
 
   // Privates
   // --------
+  var KeyCode = {
```

Comment:
Maybe stick this in domUtil, so it can be shared by all code that needs it? It's pretty useful.